Documentation/network/bgp-control-plane/bgp-control-plane-v2.rst

```diff
@@ -736,6 +736,63 @@ Similarly, ``internalTrafficPolicy`` is considered for ``ClusterIP`` advertiseme
     and ignores the configuration of ``.spec.internalTrafficPolicy``.
 
 
+Overlapping Advertisements
```

Comment:
@qmonnet and @harsimran-pabla Could you two look over the Docs update again?

Here's what it looks like on my end:

<img width="1071" alt="Screenshot 2024-12-12 at 9 54 45 PM" src="https://github.com/user-attachments/assets/05b5669f-26b3-4fe6-8546-842361026495" />

Reply:
Looks good to me now, thank you!